File: hackerrank/algorithms/simple-array-sum.c
// https://www.hackerrank.com/challenges/simple-array-sum
#include <stdio.h>

int main() {
	int n, x, s = 0;
	scanf("%d\n", &n);
	for (int i = 0; i < n; i++) {
		scanf("%d\n", &x);
		s += x;
	}
	printf("%d\n", s);
}

File: hackerrank/algorithms/simple-array-sum.ml
(* https://www.hackerrank.com/challenges/simple-array-sum *)
open Core_kernel

let () =
  read_int () |> ignore;
  read_line ()
  |> String.split ~on:' '
  |> List.map Int.of_string
  |> List.fold_left (+) 0
  |> Printf.printf "%d\n"
